### DISCUSSION TITLE
DropDownList

### DISCUSSION THREAD

**Discussion by AliKantouar** [jferard/fastods#222] [General]
Hello !
First of all thank you for the work you share here it's very useful in my case. About the topic of this message i was asking myself if there is a feature to create dropdown list i didn't found it in examples, is it something already in the project or it will comes later ?

**Comment by jferard** [comment]
Hi!
I believe you are talking of "Data > Validity..." and the tab "Criteria", then allow a "List" (for instance). This feature is not implemented in FastODS, but I will have a look.

---

A few notes (`content.xml`):

    <office:body>
        <office:spreadsheet>
             <table:content-validations>
                   <table:content-validation table:name="val1" ... >
             ...

And then:

    <table:table-cell table:content-validation-name="val1" ... />

**Comment by jferard** [comment]
@AliKantouar Can you have a look at <stroke>https://github.com/jferard/fastods/blob/master/fastods-examples/src/main/java/com/github/jferard/fastods/examples/J_Misc.java#L495</stroke>. That's a first draft. Does it meet your needs?

**EDIT: Was moved to https://github.com/jferard/fastods/blob/master/fastods-examples/src/main/java/com/github/jferard/fastods/examples/M_Protection.java#L70**

**Reply by jferard** [reply]
Link to the tutorial: https://github.com/jferard/fastods/wiki/Tutorial#validate-the-content-of-a-cell

**Reply by AliKantouar** [reply]
Hi ! I just tested it with the tutorial and it works great ! It was exactly what i was thinking thanks for your reactivity
